bot.rb

```diff
@@ -185,6 +192,7 @@ def should_mark_as_merged?(pr)
 
     def notify_action_channel_about(needs_attention_prs)
       return unless needs_attention_prs.any?
+      return unless Bot.should_notify_slack_message_not_that_important_though?
```

Comment:
Nitpicking, I'd put this on the first line to be consistent with the first call.